Serverless changelog Dec 2, 2025 (#4210)
elastic/docs-content: release-notes/elastic-cloud-serverless/index.md

# {{serverless-full}} changelog [elastic-cloud-serverless-changelog]
Review the changes, fixes, and more to {{serverless-full}}.

## December 2, 2025 [serverless-changelog-12022025]

### Features and enhancements [serverless-changelog-12022025-features-enhancements]

> * Adds new fields to the Schedule exports flyout form for email notification: Cc, Bcc, and Subject and Message fields with Mustache templating support [#242922]({{kib-pull}}242922)

Reviewer: ```suggestion
* Adds Cc, Bcc, Subject, and Message fields with Mustache templating support to the Schedule exports flyout for email notifications  [#242922]({{kib-pull}}242922)
```

* Allows users to enable scheduled reports [#244202]({{kib-pull}}244202)
* Adds a background {{fleet}} policy revisions cleanup task to automatically remove excess policy revisions from the `.fleet-policies` index [#242612]({{kib-pull}}242612)
* Automatically migrates component template ILM policies during setup [#243333]({{kib-pull}}243333)
* Improves suggestion ordering using categorization to provide more relevant results [#243312]({{kib-pull}}243312)
* Allows selecting the column type in the lookup index editor [#241637]({{kib-pull}}241637)

> * Adds the **Read global parameters** sub-feature privilege which allows users to read the values of synthetics global parameters [#243821]({{kib-pull}}243821)

Reviewer: ```suggestion
* Adds the **Read global parameters** sub-feature privilege which allows you to read the values of synthetics global parameters [#243821]({{kib-pull}}243821)
```
Maybe put this one higher up? Seems important

* Ensures that infrastructure inventory UIs accurately reflect supported schemas [#244481]({{kib-pull}}244481)
* Adds a warning when deleting API keys currently in use by alerting rules [#243353]({{kib-pull}}243353)
* Removes the median line length check in the categorization anomaly detection job [#243827]({{kib-pull}}243827)
* Allows filtering alerts using the KQL search bar [#240100]({{kib-pull}}240100)
* Introduces the Attachments API for Streams [#243597]({{kib-pull}}243597)
* Introduces new UI components for the Drop processor [#243131]({{kib-pull}}243131)
* Adds SLO support for Streams attachments, migrates the UI to use the Attachments API for dashboards, rules, and SLOs, and removes deprecated API endpoints [#244092]({{kib-pull}}244092)
* Allows adding custom descriptions for enrichment processors [#243998]({{kib-pull}}243998)
* Prevents conflicting actions in the Partitioning tab [#244228]({{kib-pull}}244228)
* Improves handling of missing streams [#244366]({{kib-pull}}244366)

> * Allow users to configure the visibility of the Streams app per space [#244285]({{kib-pull}}244285)

Author: ```suggestion
* Allows you to configure the visibility of the Streams app per space [#244285]({{kib-pull}}244285)
```

* Allows users to enable scheduled reports [#244202]({{kib-pull}}244202)
* Adds a background {{fleet}} policy revisions cleanup task to automatically remove excess policy revisions from the `.fleet-policies` index [#242612]({{kib-pull}}242612)
* Automatically migrates component template ILM policies during setup [#243333]({{kib-pull}}243333)
* Improves suggestion ordering using categorization to provide more relevant results [#243312]({{kib-pull}}243312)
* Allows you to select a column type in the lookup index editor [#241637]({{kib-pull}}241637)
* Ensures that infrastructure inventory UIs accurately reflect supported schemas [#244481]({{kib-pull}}244481)
* Adds a warning when deleting API keys currently in use by alerting rules [#243353]({{kib-pull}}243353)
* Removes the median line length check in the categorization anomaly detection job [#243827]({{kib-pull}}243827)
* Allows you to filter alerts using the KQL search bar [#240100]({{kib-pull}}240100)
* Introduces the Attachments API for Streams [#243597]({{kib-pull}}243597)
* Introduces new UI components for the Drop processor [#243131]({{kib-pull}}243131)

> * Adds service-level objective (SLO) support for Streams attachments, migrates the UI to use the Attachments API for dashboards, rules, and SLOs, and removes deprecated API endpoints [#244092]({{kib-pull}}244092)

Reviewer 2: ```suggestion
* Adds service-level objective (SLO) support for streams attachments, migrates the UI to use the Attachments API for dashboards, rules, and SLOs, and removes deprecated API endpoints [#244092]({{kib-pull}}244092)
```

* Redesigns the Lens configuration flyout to show layers as tabs instead of vertically stacked panels [#235372]({{kib-pull}}235372)
* Consolidates attachments into a single Attachments tab with sub-tab navigation [#243708]({{kib-pull}}243708)
* Adds the {{esql}} `CHUNK` function in technical preview [#138621]({{es-pull}}138621)
* Improves support for the `first()` and `last()` aggregation functions in {{esql}} by disabling vector dispatch for blocks [#138390]({{es-pull}}138390)
* Adds informative timestamps to async {{esql}} query results [#137957]({{es-pull}}137957)
* Add Groq as a chat completion inference service for {{ml}} [#138251]({{es-pull}}138251)
* Adds the node-scoped `vectors.indexing.use_gpu` setting to control GPU usage for vector indexing [#138738]({{es-pull}}138738)
* Adds routing support to the `_project/tags` endpoint
* Allows point-in-time (PIT) searches to span multiple projects [#137966]({{es-pull}}137966)
* Excludes synthetic `_id` postings from disk usage statistics [#138745]({{es-pull}}138745)
* Allows `project_routing` to be specified as a query parameter in EQL requests [#138559]({{es-pull}}138559)

> * Avoids retrieving unnecessary fields  during the node-reduce phase in {{esql}} queries [#137920]({{es-pull}}137920)

Reviewer 2: ```suggestion
* Avoids retrieving unnecessary fields during the node-reduce phase in {{esql}} queries [#137920]({{es-pull}}137920)
```
